integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -246,8 +248,13 @@ public int create(String path, @mode_t long mode, FuseFileInfo fi) {
             MAX_OPEN_FILES);
         return -ErrorCodes.EMFILE();
       }
-
-      FileOutStream os = mFileSystem.createFile(uri);
+      PMode pmode = PMode.newBuilder()
+          .setOwnerBits(PMode.parseOwnerBits(mode))
+          .setGroupBits(PMode.parseGroupBits(mode))
+          .setOtherBits(PMode.parseOtherBits(mode))
+          .build();
+      FileOutStream os = mFileSystem.createFile(uri,
+          CreateFilePOptions.newBuilder().setMode(pmode).build());
```

Comment:
PMode is an auto-generated proto file that we are not supposed to touch it. There is an example of how to transfer the input `long mode` to Alluxio `PMode` in https://github.com/Alluxio/alluxio/blob/5f9be23716ef09e2e303931674c66163d613ab58/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java#L153

Reply:
re.
In addition, if any static helper methods are needed, it is better to put the helper functions into `alluxio/security/authorization/Mode.java`

Reply:
Good idea. Fixed it.